libraries/Microsoft.Bot.Streaming/Payloads/Assemblers/ReceiveResponseAssembler.cs

```diff
@@ -73,15 +73,15 @@ public void OnReceive(Header header, Stream stream, int contentLength)
                 stream.Position = 0;
 
                 // Execute the request on a seperate Task
-                Background.Run(() => ProcessResponse(stream));
+                Background.Run(() => ProcessResponseAsync(stream));
             }
 
             // else: still receiving data into the stream
         }
 
         public void Close() => _streamManager.CloseStream(Id);
 
-        private async Task ProcessResponse(Stream stream)
+        private async Task ProcessResponseAsync(Stream stream)
```

Comment:
>ProcessResponseAsync [](start = 27, length = 20)

cancelationToken

Reply:
Tried by couldn't. We don't have cancellationToken in the callers to this private method (adding it would break binary compat)